opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -243,6 +243,8 @@ namespace Opm {
                 residual_norms_history_.clear();
                 current_relaxation_ = 1.0;
                 dx_old_ = 0.0;
+                convertInput( iteration, reservoir_state, ebosSimulator_ );
+                ebosSimulator_.model().invalidateIntensiveQuantitiesCache(/*timeIdx=*/0);
             }
```

Comment:
Maybe this should be moved to where you actually do the restart. Then it should only effect restarts.